src/test/ui/pattern/usefulness/match-byte-array-patterns-2.stderr

```diff
@@ -7,11 +7,11 @@ LL |     match buf {
    = help: ensure that all possible cases are being handled, possibly by adding wildcards or more match arms
    = note: the matched value is of type `&[u8; 4]`
 
-error[E0004]: non-exhaustive patterns: `&[]`, `&[_]`, `&[_, _]` and 2 more not covered
+error[E0004]: non-exhaustive patterns: `&[0u8..=64u8, _, _, _]` and `&[66u8..=std::u8::MAX, _, _, _]` not covered
```

Comment:
some regression or improvement in diagnostics depending on your view. It's more precise, because u8 arrays behind references were always treated just like byte strings and the elements were essentially ignored.